src/components/Sidebar/SidebarItem.tsx

```diff
@@ -93,31 +42,10 @@ const SidebarItemName = styled.h4`
   color: #1c1c1e;
 `;
 
-const SettingsItemWrapper = styled.li`
-  display: flex;
-  padding: 16px 0 16px 80px;
-  gap: 32px;
-  align-items: center;
-  cursor: pointer;
-  &:hover {
-    background-color: rgba(79, 39, 191, 0.1);
-  }
-  &:first-of-type {
-    padding-top: 12px;
-  }
-  &:last-of-type {
-    padding-bottom: 20px;
-  }
-`;
-
-const ItemsWrapper = styled.div`
-  display: flex;
-  flex-direction: column;
-`;
-
-const SettingsWrapper = styled.div`
+const ItemsWrapper = styled.div<{paddingLeft: string}>`
   display: flex;
   flex-direction: column;
+  padding-left: ${({paddingLeft}) => paddingLeft};
```

Comment:
Make paddingLeft: number and add px here.
If paddingLeft is string, then I can write paddingLeft="kek" and it will break the styles.
But if it's number, then it will always be pixels. No error.